uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/components/staff-leave-report.html

```diff
@@ -48,8 +48,9 @@
                   </td>
                   <td></td>
                   <td></td>
-                  <td ng-repeat="absenceType in report.absenceTypesFiltered track by $index">
-                    {{absenceType.entitlement}}
+                  <td ng-repeat="absenceType in report.absenceTypesFiltered track by $index"
+                    title="({{ absenceType.calculation_unit_label | lowercase }})">
+                    {{+absenceType.entitlement + absenceType.calculation_unit_symbol}}
```

Comment:
I don't think the + for `absenceType.entitlement` is necessary. I'd remove it if it's not because it makes me think I'm trying to add two numbers together.

Reply:
Please check other instances of this. You are adding two strings together, no need to convert one of them to a number to then convert them back to a string.

Reply:
`+` was needed to transform `0.00` to `0` so I doesn't look like `0.00h` which is just weird. However, now there is a new shiny filter for this, so all `+` have gone!